source/extensions/filters/http/ext_proc/processor_state.cc

```diff
@@ -91,7 +91,13 @@ absl::Status ProcessorState::handleHeadersResponse(const HeadersResponse& respon
         return mut_status;
       }
     }
-    if (common_response.clear_route_cache()) {
+
+    bool allow_clearing_route_cache =
```

Comment:
This flag should be available when this class is constructed and its value is consistent.

Can we add a member variable for this and only init it in constructor? so that we can avoid retrieving it every time when this function is called, which is redundant

Reply:
Moved it to the filter config. Followed the convention there.